README.md

```diff
@@ -165,7 +165,7 @@ Hurray! You successfully made a contribution!
 
 - ### **J**
 
-  - []()
+  - [Jatin Kumar](https://github.com/Jatinkumar30)
```

Comment:
Your PR looks good.
@Jatinkumar30 Thanks for making your contribution to the open-source world and stepping your feet in the open-source community.
Happy Hacking.! 👍 💯